### PR TITLE
Fixed layout

### PR DESCRIPTION
**Pull request by ImieboGodson**
- Added Wrapper component that holds header and footer then loads routes dynamically

### CODE REVIEW

File: frontend/src/utils/Wrapper.jsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import Navbar from "../components/Navbar"
+import Footer from "../components/footer"
+import RoutesComponent from "../routes/RoutesComponent"
+
+const Wrapper = () => {
```
Comment (MorganJay) [inline frontend/src/utils/Wrapper.jsx]: I don't know if this Wrapper needs to be extracted into its own component, this adds more complexity. you can just import the Navbar and Footer in the App component

Comment (MorganJay) [reply frontend/src/utils/Wrapper.jsx]: Then again the Navbar and Footer are rendered conditionally across some other screens of the whole web app, so you might want to prepare for that now

**Review by MorganJay — CHANGES_REQUESTED** [review]

**Comment by ImieboGodson** [comment]
@MorganJay  see current  change according to your last review

File: frontend/src/App.js [hunk 2]
```diff
@@ -1,10 +1,31 @@
...
+
+  
+
   return (
     <>
-      <RoutesComponent />
+      <Navbar />
+        <RoutesComponent />
+      {
+        (route === '' || route === 'FAQs' || route === 'aboutUs' || route === 'support')
```
Comment (MorganJay) [inline frontend/src/App.js]: a switch statement would look cleaner and more readable here

**Review by MorganJay — CHANGES_REQUESTED** [review]

**Comment by ImieboGodson** [comment]
@MorganJay Kindly find changes as requested.

**Pull request merged**